utility/src/main/java/org/oppia/util/parser/ImageLoaderModule.kt

```diff
@@ -0,0 +1,17 @@
+package org.oppia.util.parser
+
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+/** Provides image loading dependencies. */
+@Module
+class ImageLoaderModule {
```

Comment:
Optional, consider naming this module & the file ``GlideImageLoaderModule`` since it's really providing the Glide implementation of ``ImageLoader``.

Reply:
renamed to GlideImageLoader.